desktop/cmp/button/ButtonGroup.ts

```diff
@@ -23,6 +24,9 @@ export interface ButtonGroupProps<M extends HoistModel = null>
 
     /** True to render in a vertical orientation. */
     vertical?: boolean;
+
+    /** Override non-optional `children` from BpButtonGroupProps to support ElementFactor. */
+    children?: ReactNode | undefined;
```

Comment:
Why " | undefined" you ask?

Because that is the pattern used for declaring optional children in the react d.ts node_modules/@types/react/index.d.ts